src/xo-app/xoa-updates/index.js

```diff
@@ -129,6 +135,7 @@ export default class XoaUpdates extends Component {
     serverVersion.then(serverVersion => {
       this.setState({ serverVersion })
     })
+    update()
```

Comment:
No, update just checks for manifest, only upgrade has side-effects (like the apt vocabulary).

The xoa-updater client updates every hour, and I added that so a visit of the update page refreshes information.